dashboard/components/BackPressureTableWithoutPrometheus.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 RisingWave Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+    Table,
+    TableCaption,
+    TableContainer,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+} from "@chakra-ui/react"
+import { sortBy } from "lodash"
+import Head from "next/head"
+import { Fragment, useEffect, useState } from "react"
+import useErrorToast from "../hook/useErrorToast"
+import RateBar from "./RateBar"
+import { BackPressureInfo, getBackPressureWithoutPrometheus, BackPressuresMetrics, calculateBPRate, INTERVAL } from "../pages/api/metric"
+
+export default function BackPressureTableWithoutPrometheus({
```

Comment:
Is this still used?

Reply:
This function is not used currently. I have one question related to it. I check the previous implememntation. For example: https://github.com/risingwavelabs/risingwave/blob/8cb91531cb1e8a9a49d835aacd4fad441874ba2f/dashboard/components/BackPressureTable.tsx#L38
Because I guess my new API should be same with the provious back pressure API.I find it only used here which is already comment out: https://github.com/risingwavelabs/risingwave/blob/8cb91531cb1e8a9a49d835aacd4fad441874ba2f/dashboard/components/FragmentGraph.tsx#L482

So I guess it will used in the future, then I impelement it here. If you think it will be useless in the future, I can delete it.

Reply:
`BackPressureTable` was used in the initial version, back pressure information was displayed in the form of a table. It will no longer be used in the future and can be deleted directly.

Reply:
Will delete the `BackPressureTable` and `BackPressureTableWithoutPrometheus`.